setup.cfg

```diff
@@ -1,3 +0,0 @@
-[flake8]
-exclude = .gitignore,venv
```

Comment:
Just checking that this is alright to `exclude` too? I don't find any linter errors with it gone, even with my `.venv` so it seems okay to me

Reply:
Actually Flake8 does not exclude the virtual env folders, the dev must exclude them manually based on what they named the folders `.env` `.venv` `venv` etc.. 

This is why the readme informs the users to use `flake8 *.py && flake8 listeners/` in order to lint the project